front/etf-recommendation/lib/api/etf.ts

```diff
@@ -16,6 +16,14 @@ export interface EtfReturnDto {
   returnRate: number;
 }
 
+export interface EtfAllResponse {
+  totalPage: number;
+  totalCount: number;
+  currentPage: number;
+  pageSize: number;
+  etfReadResponseList: EtfReturnDto[];
+}
```

Comment:
`totalPage`, `currentPage`, `pageSize`는 빼야겠네요
```suggestion
export interface EtfAllResponse {
  totalCount: number;
  etfReadResponseList: EtfReturnDto[];
}
```